### DISCUSSION TITLE
--read-only mount seen by system as 'rw'

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#2136] [General]
Not really a 'bug', but a `--read-only` mount is seen by system (command `mount`) as `rw`.

`JuiceFS:jfs on /mnt/jfs type fuse.juicefs (rw,relatime,user_id=0,group_id=0,default_permissions,allow_other)`

**Comment by davies** [comment]
We used to mount it with `ro`, then we can't open the special `.control` for write (used by warmup)
